src/current/v25.3/ldap-authentication.md

```diff
@@ -36,14 +44,15 @@ Because CockroachDB maintains no more than one LDAP connection per node, for a c
 - Network connectivity on port 636 for LDAPS.
 - A service account (bind DN) with permissions to search the directory for basic information about users and groups. For example, in Microsoft Entra ID, a [service principal](https://learn.microsoft.com/entra/architecture/secure-service-accounts) with the Directory Readers role.
 - The LDAP server's CA certificate, if using a custom CA not already trusted by the CockroachDB host.
+- Verification that the attribute values that will become CockroachDB usernames meet the CockroachDB [requirements for usernames]({% link {{ page.version.version }}/create-user.md %}#user-names).
 
 Before you begin, it may be useful to enable authentication logging, which can help you confirm successful configuration or troubleshoot issues. For details, refer to [Troubleshooting](#troubleshooting).
 
 ### Step 1: Enable redaction of sensitive cluster settings
 
-You will set LDAP bind credentials for the service account that enables this integration using the cluster setting `server.host_based_authentication.configuration`. You will also configure the mapping of external identities to CockroachDB SQL users using the cluster settings `server.identity_map.configuration`.
+For this integration, you will need to store LDAP bind credentials for the service account that enables the integration in the cluster setting `server.host_based_authentication.configuration`. You will also configure the mapping of external identities to CockroachDB SQL users with the cluster setting `server.identity_map.configuration`. In addition, for a custom CA configuration, you may need to store certificate and key details in specified cluster settings. 
```

Comment:
Suggest linking directly to both cluster settings on cluster-settings.md.

Is it clear what "specified cluster settings" refers to?